Taskfile.yml

```diff
@@ -149,14 +150,29 @@ tasks:
       - TF_REATTACH_PROVIDERS={{.TF_REATTACH_PROVIDERS}} terraform -chdir="{{.WORKSPACE_DIR}}" plan -var="OPSLEVEL_API_TOKEN=$OPSLEVEL_API_TOKEN"
 
   terraform-clean:
-    desc: Completely wipe terraform state and all terraform generated files from "{{.WORKSPACE_DIR}}"
+    desc: Completely wipe terraform state, terraform generated files, and local OpsLevel provider binaries
     aliases: ["clean"]
-    prompt: Remove '*.tfstate .terraform.lock.hcl ./terraform.* .terraform/**' from '{{.WORKSPACE_DIR}}' directory?
+    prompt: Remove all '*.tfstate .terraform.lock.hcl ./terraform.* .terraform/**' and OpsLevel binaries from plugins directory?
     cmds:
-      - task: terraform-rm-local-provider-no-prompt
+      - rm -rf {{.PLUGIN_DIR}}
+      - task: terraform-clean-from
         vars: { CLEAN_DIR: "{{.WORKSPACE_DIR}}" }
-      - cmd: rm -rf {{.WORKSPACE_DIR}}/*.tfstate*
+      - task: terraform-clean-from
+        vars: { CLEAN_DIR: "{{.TEST_DIR}}" }
+      - task: terraform-clean-from
+        vars: { CLEAN_DIR: "{{.TEST_DIR}}/local" }
+      - task: terraform-clean-from
+        vars: { CLEAN_DIR: "{{.TEST_DIR}}/remote" }
+
+  terraform-clean-from:
+    internal: true
+    cmds:
+      - task: terraform-rm-local-provider-no-prompt
```

Comment:
doesn't this mean you are running this several times given the above changes to `terraform-clean`

Reply:
yes

Reply:
consolidated a lot of the unnecessary redirection